README.md

```diff
@@ -1,11 +1,23 @@
-<img src="https://media2.giphy.com/media/Nx0rz3jtxtEre/giphy.gif?cid=ecf05e47k1el00v2wbzxdaemqy93cxosmaf6bp2r9trzlc03&rid=giphy.gif&ct=g"/>
+# Sam's Notes Application (that doubles as an "About Me")
 
-If you're looking for a job, head on over to [PROMPT.md](./PROMPT.md).
+## How go we get this thing going?
 
+-   open up two terminals
+-   `cd` into the `client` in one and the `server` in the other
 
-</br></br></br></br>
+In both the `server` terminal and the `client` terminal, run the following commands:
 
-<!-- Delete this line and everything above it -->
+```
+npm install
+npm start
+```
 
-# `Your App Name Here`
-Add some pretty documentation here that tells us how to start your app so we can see the awesome thing you built!
+This will spin up the `server` on `localhost:5000` and the `client` on `localhost:3000`
```

Comment:
So fun fact that I just discovered; [this port is apparently used by macOS on Monterrey](https://developer.apple.com/forums/thread/682332) 😅 

This definitely isn't something I would have expected you to discover, but one thing that can be helpful for other devs is to consume the `PORT` from a `.env` file and simply default to something (`5000` in your case) so they can quickly override it 🎉